main.go

```diff
@@ -37,6 +37,7 @@ import (
 
 	//+kubebuilder:scaffold:imports
 	consolev1 "github.com/openshift/api/console/v1"
+	console "github.com/red-hat-storage/odf-operator"
```

Comment:
```suggestion
	console "github.com/red-hat-data-services/odf-operator/console"
```

Reply:
Don't change to `data-services`, we've just moved away from that. But yet to the `console` addition.